packages/casl-angular/package.json

```diff
@@ -46,9 +46,9 @@
   "author": "Sergii Stotskyi <sergiy.stotskiy@gmail.com>",
   "license": "MIT",
   "peerDependencies": {
-    "@angular/core": "^12.0.0",
+    "@angular/core": "^12.0.0 || ^13.0.0-0",
```

Comment:
I'm not sure it works properly with Angular v13. You need to update dev deps  angular version to ensure that it's true